src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -693,6 +693,23 @@ class genUtils {
 
         return result;
     }
+    
+    static scrollToElement(element: Element) {
+        const containerSelector = ".js-scroll-container";
```

Comment:
can we move this as second parameter with default value?

Reply:
done